crates/symbolicator-service/tests/integration/snapshots/integration__sourcemap__source_expansion.snap

```diff
@@ -4,7 +4,7 @@ expression: response.unwrap()
 ---
 stacktraces:
   - frames:
-      - status: symbolicated
+      - status: missing_sourcemap
```

Comment:
I'm not sure about this error tbh. In original tests, we simply applied source context to the frames, and that's it. No errors, no `raw_stacktrace` (as no source maps were applied, thus we only kept around original frames).